src/Persistence/Array_.php

```diff
@@ -37,9 +37,51 @@ public function __construct(array $data = [])
      *             - https://github.com/atk4/data/blob/90ab68ac063b8fc2c72dcd66115f1bd3f70a3a92/src/Reference/ContainsMany.php#L66
      *             remove once fixed/no longer needed
      */
-    public function getRawDataByTable(string $table): array
+    public function getRawDataByTable(Model $model, string $table): array
```

Comment:
I would only remove second argument and use $model->table instead

Reply:
![image](https://user-images.githubusercontent.com/2228672/92313210-780ffc80-efc9-11ea-885d-4bdbe05bc7a5.png)

@georgehristov please see, the table names are different, models in Reference/Join seems to use some table_alias different than table, this was one of the initial issue with this PR